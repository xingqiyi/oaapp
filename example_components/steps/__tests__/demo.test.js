import { webDemoTest } from '../../../tests/shared/demoTest';

webDemoTest('steps');
