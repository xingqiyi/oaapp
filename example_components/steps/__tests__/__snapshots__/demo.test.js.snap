// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./components/steps/demo/basic.md correctly 1`] = `
<div
  class="am-wingblank am-wingblank-lg"
>
  <div
    class="sub-title"
  >
    Small size
  </div>
  <div
    class="am-whitespace am-whitespace-md"
  />
  <div
    class="am-steps am-steps-small am-steps-vertical am-steps-hidden"
  >
    <div
      class="am-steps-item am-steps-status-finish am-steps-custom"
    >
      <div
        class="am-steps-tail"
        style="padding-right:0;"
      >
        <i />
      </div>
      <div
        class="am-steps-step"
      >
        <div
          class="am-steps-head"
        >
          <div
            class="am-steps-head-inner"
          >
            <span
              class="am-steps-icon"
            >
              <svg
                class="am-icon am-icon-check-circle-o am-icon-md"
              >
                <use
                  xlink:href="#check-circle-o"
                />
              </svg>
            </span>
          </div>
        </div>
        <div
          class="am-steps-main"
        >
          <div
            class="am-steps-title"
          >
            Finished
          </div>
          <div
            class="am-steps-description"
          >
            This is description
          </div>
        </div>
      </div>
    </div>
    <div
      class="am-steps-item am-steps-status-process"
    >
      <div
        class="am-steps-tail"
        style="padding-right:0;"
      >
        <i />
      </div>
      <div
        class="am-steps-step"
      >
        <div
          class="am-steps-head"
        >
          <div
            class="am-steps-head-inner"
          >
            <span
              class="am-steps-icon"
            >
              2
            </span>
          </div>
        </div>
        <div
          class="am-steps-main"
        >
          <div
            class="am-steps-title"
          >
            In Progress
          </div>
          <div
            class="am-steps-description"
          >
            This is description
          </div>
        </div>
      </div>
    </div>
    <div
      class="am-steps-item am-steps-status-wait am-steps-custom ellipsis-item"
    >
      <div
        class="am-steps-tail"
        style="padding-right:0;"
      >
        <i />
      </div>
      <div
        class="am-steps-step"
      >
        <div
          class="am-steps-head"
        >
          <div
            class="am-steps-head-inner"
          >
            <span
              class="am-steps-icon"
            >
              <svg
                class="am-icon am-icon-ellipsis am-icon-md"
              >
                <use
                  xlink:href="#ellipsis"
                />
              </svg>
            </span>
          </div>
        </div>
        <div
          class="am-steps-main"
        >
          <div
            class="am-steps-title"
          >
            Waiting
          </div>
          <div
            class="am-steps-description"
          >
            This is description
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="sub-title"
  >
    Small size, single line text
  </div>
  <div
    class="am-whitespace am-whitespace-md"
  />
  <div
    class="am-steps am-steps-small am-steps-vertical am-steps-hidden"
  >
    <div
      class="am-steps-item am-steps-status-finish am-steps-custom"
    >
      <div
        class="am-steps-tail"
        style="padding-right:0;"
      >
        <i />
      </div>
      <div
        class="am-steps-step"
      >
        <div
          class="am-steps-head"
        >
          <div
            class="am-steps-head-inner"
          >
            <span
              class="am-steps-icon"
            >
              <svg
                class="am-icon am-icon-check-circle-o am-icon-md"
              >
                <use
                  xlink:href="#check-circle-o"
                />
              </svg>
            </span>
          </div>
        </div>
        <div
          class="am-steps-main"
        >
          <div
            class="am-steps-title"
          >
            Finished
          </div>
        </div>
      </div>
    </div>
    <div
      class="am-steps-item am-steps-status-process"
    >
      <div
        class="am-steps-tail"
        style="padding-right:0;"
      >
        <i />
      </div>
      <div
        class="am-steps-step"
      >
        <div
          class="am-steps-head"
        >
          <div
            class="am-steps-head-inner"
          >
            <span
              class="am-steps-icon"
            >
              2
            </span>
          </div>
        </div>
        <div
          class="am-steps-main"
        >
          <div
            class="am-steps-title"
          >
            In Progress
          </div>
        </div>
      </div>
    </div>
    <div
      class="am-steps-item am-steps-status-wait am-steps-custom ellipsis-item"
    >
      <div
        class="am-steps-tail"
        style="padding-right:0;"
      >
        <i />
      </div>
      <div
        class="am-steps-step"
      >
        <div
          class="am-steps-head"
        >
          <div
            class="am-steps-head-inner"
          >
            <span
              class="am-steps-icon"
            >
              <svg
                class="am-icon am-icon-ellipsis am-icon-md"
              >
                <use
                  xlink:href="#ellipsis"
                />
              </svg>
            </span>
          </div>
        </div>
        <div
          class="am-steps-main"
        >
          <div
            class="am-steps-title"
          >
            Waiting
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="sub-title"
  >
    Default size
  </div>
  <div
    class="am-whitespace am-whitespace-lg"
  />
  <div
    class="am-steps am-steps-vertical am-steps-hidden"
  >
    <div
      class="am-steps-item am-steps-status-process error-tail"
    >
      <div
        class="am-steps-tail"
        style="padding-right:0;"
      >
        <i />
      </div>
      <div
        class="am-steps-step"
      >
        <div
          class="am-steps-head"
        >
          <div
            class="am-steps-head-inner"
          >
            <span
              class="am-steps-icon"
            >
              1
            </span>
          </div>
        </div>
        <div
          class="am-steps-main"
        >
          <div
            class="am-steps-title"
          >
            Finished
          </div>
          <div
            class="am-steps-description"
          >
            This is description
          </div>
        </div>
      </div>
    </div>
    <div
      class="am-steps-item am-steps-status-error am-steps-custom ellipsis-item"
    >
      <div
        class="am-steps-tail"
        style="padding-right:0;"
      >
        <i />
      </div>
      <div
        class="am-steps-step"
      >
        <div
          class="am-steps-head"
        >
          <div
            class="am-steps-head-inner"
          >
            <span
              class="am-steps-icon"
            >
              <svg
                class="am-icon am-icon-cross-circle-o am-icon-md"
              >
                <use
                  xlink:href="#cross-circle-o"
                />
              </svg>
            </span>
          </div>
        </div>
        <div
          class="am-steps-main"
        >
          <div
            class="am-steps-title"
          >
            Error
          </div>
          <div
            class="am-steps-description"
          >
            This is description
          </div>
        </div>
      </div>
    </div>
    <div
      class="am-steps-item am-steps-status-wait am-steps-custom ellipsis-item"
    >
      <div
        class="am-steps-tail"
        style="padding-right:0;"
      >
        <i />
      </div>
      <div
        class="am-steps-step"
      >
        <div
          class="am-steps-head"
        >
          <div
            class="am-steps-head-inner"
          >
            <span
              class="am-steps-icon"
            >
              <svg
                class="am-icon am-icon-ellipsis am-icon-md"
              >
                <use
                  xlink:href="#ellipsis"
                />
              </svg>
            </span>
          </div>
        </div>
        <div
          class="am-steps-main"
        >
          <div
            class="am-steps-title"
          >
            Waiting
          </div>
          <div
            class="am-steps-description"
          >
            This is description
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="sub-title"
  >
    Customized status 
  </div>
  <div
    class="am-whitespace am-whitespace-lg"
  />
  <div
    class="am-steps am-steps-vertical am-steps-hidden"
  >
    <div
      class="am-steps-item am-steps-status-finish am-steps-custom"
    >
      <div
        class="am-steps-tail"
        style="padding-right:0;"
      >
        <i />
      </div>
      <div
        class="am-steps-step"
      >
        <div
          class="am-steps-head"
        >
          <div
            class="am-steps-head-inner"
          >
            <span
              class="am-steps-icon"
            >
              <svg
                class="am-icon am-icon-pay-circle am-icon-md"
              >
                <use
                  xlink:href="#pay-circle"
                />
              </svg>
            </span>
          </div>
        </div>
        <div
          class="am-steps-main"
        >
          <div
            class="am-steps-title"
          >
            Step 1
          </div>
        </div>
      </div>
    </div>
    <div
      class="am-steps-item am-steps-status-process am-steps-custom error-tail"
    >
      <div
        class="am-steps-tail"
        style="padding-right:0;"
      >
        <i />
      </div>
      <div
        class="am-steps-step"
      >
        <div
          class="am-steps-head"
        >
          <div
            class="am-steps-head-inner"
          >
            <span
              class="am-steps-icon"
            >
              <svg
                class="am-icon am-icon-pay-circle am-icon-md"
              >
                <use
                  xlink:href="#pay-circle"
                />
              </svg>
            </span>
          </div>
        </div>
        <div
          class="am-steps-main"
        >
          <div
            class="am-steps-title"
          >
            Step 2
          </div>
        </div>
      </div>
    </div>
    <div
      class="am-steps-item am-steps-status-error am-steps-custom"
    >
      <div
        class="am-steps-tail"
        style="padding-right:0;"
      >
        <i />
      </div>
      <div
        class="am-steps-step"
      >
        <div
          class="am-steps-head"
        >
          <div
            class="am-steps-head-inner"
          >
            <span
              class="am-steps-icon"
            >
              <svg
                class="am-icon am-icon-pay-circle am-icon-md"
              >
                <use
                  xlink:href="#pay-circle"
                />
              </svg>
            </span>
          </div>
        </div>
        <div
          class="am-steps-main"
        >
          <div
            class="am-steps-title"
          >
            Step 3
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="sub-title"
  >
    Customized icon 
  </div>
  <div
    class="am-whitespace am-whitespace-lg"
  />
  <div
    class="am-steps am-steps-vertical am-steps-hidden"
  >
    <div
      class="am-steps-item am-steps-status-finish am-steps-custom"
    >
      <div
        class="am-steps-tail"
        style="padding-right:0;"
      >
        <i />
      </div>
      <div
        class="am-steps-step"
      >
        <div
          class="am-steps-head"
        >
          <div
            class="am-steps-head-inner"
          >
            <span
              class="am-steps-icon"
            >
              <svg
                class="am-icon am-icon-pay-circle am-icon-md"
              >
                <use
                  xlink:href="#pay-circle"
                />
              </svg>
            </span>
          </div>
        </div>
        <div
          class="am-steps-main"
        >
          <div
            class="am-steps-title"
          >
            Step 1
          </div>
          <div
            class="am-steps-description"
          >
            This is description
          </div>
        </div>
      </div>
    </div>
    <div
      class="am-steps-item am-steps-status-process am-steps-custom"
    >
      <div
        class="am-steps-tail"
        style="padding-right:0;"
      >
        <i />
      </div>
      <div
        class="am-steps-step"
      >
        <div
          class="am-steps-head"
        >
          <div
            class="am-steps-head-inner"
          >
            <span
              class="am-steps-icon"
            >
              <svg
                class="am-icon am-icon-pay-circle am-icon-md"
              >
                <use
                  xlink:href="#pay-circle"
                />
              </svg>
            </span>
          </div>
        </div>
        <div
          class="am-steps-main"
        >
          <div
            class="am-steps-title"
          >
            Step 2
          </div>
          <div
            class="am-steps-description"
          >
            This is description
          </div>
        </div>
      </div>
    </div>
    <div
      class="am-steps-item am-steps-status-wait am-steps-custom"
    >
      <div
        class="am-steps-tail"
        style="padding-right:0;"
      >
        <i />
      </div>
      <div
        class="am-steps-step"
      >
        <div
          class="am-steps-head"
        >
          <div
            class="am-steps-head-inner"
          >
            <span
              class="am-steps-icon"
            >
              <svg
                class="am-icon am-icon-pay-circle am-icon-md"
              >
                <use
                  xlink:href="#pay-circle"
                />
              </svg>
            </span>
          </div>
        </div>
        <div
          class="am-steps-main"
        >
          <div
            class="am-steps-title"
          >
            Step 3
          </div>
          <div
            class="am-steps-description"
          >
            This is description
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="sub-title"
  >
    Multiple steps 
  </div>
  <div
    class="am-whitespace am-whitespace-lg"
  />
  <div
    class="am-steps am-steps-vertical am-steps-hidden"
  >
    <div
      class="am-steps-item am-steps-status-finish am-steps-custom"
    >
      <div
        class="am-steps-tail"
        style="padding-right:0;"
      >
        <i />
      </div>
      <div
        class="am-steps-step"
      >
        <div
          class="am-steps-head"
        >
          <div
            class="am-steps-head-inner"
          >
            <span
              class="am-steps-icon"
            >
              <svg
                class="am-icon am-icon-pay-circle am-icon-md"
              >
                <use
                  xlink:href="#pay-circle"
                />
              </svg>
            </span>
          </div>
        </div>
        <div
          class="am-steps-main"
        >
          <div
            class="am-steps-title"
          >
            Step 1
          </div>
        </div>
      </div>
    </div>
    <div
      class="am-steps-item am-steps-status-process am-steps-custom error-tail"
    >
      <div
        class="am-steps-tail"
        style="padding-right:0;"
      >
        <i />
      </div>
      <div
        class="am-steps-step"
      >
        <div
          class="am-steps-head"
        >
          <div
            class="am-steps-head-inner"
          >
            <span
              class="am-steps-icon"
            >
              <svg
                class="am-icon am-icon-pay-circle am-icon-md"
              >
                <use
                  xlink:href="#pay-circle"
                />
              </svg>
            </span>
          </div>
        </div>
        <div
          class="am-steps-main"
        >
          <div
            class="am-steps-title"
          >
            Step 2
          </div>
        </div>
      </div>
    </div>
    <div
      class="am-steps-item am-steps-status-error am-steps-custom"
    >
      <div
        class="am-steps-tail"
        style="padding-right:0;"
      >
        <i />
      </div>
      <div
        class="am-steps-step"
      >
        <div
          class="am-steps-head"
        >
          <div
            class="am-steps-head-inner"
          >
            <span
              class="am-steps-icon"
            >
              <svg
                class="am-icon am-icon-pay-circle am-icon-md"
              >
                <use
                  xlink:href="#pay-circle"
                />
              </svg>
            </span>
          </div>
        </div>
        <div
          class="am-steps-main"
        >
          <div
            class="am-steps-title"
          >
            Step 3
          </div>
        </div>
      </div>
    </div>
    <div
      class="am-steps-item am-steps-status-wait am-steps-custom"
    >
      <div
        class="am-steps-tail"
        style="padding-right:0;"
      >
        <i />
      </div>
      <div
        class="am-steps-step"
      >
        <div
          class="am-steps-head"
        >
          <div
            class="am-steps-head-inner"
          >
            <span
              class="am-steps-icon"
            >
              <svg
                class="am-icon am-icon-pay-circle am-icon-md"
              >
                <use
                  xlink:href="#pay-circle"
                />
              </svg>
            </span>
          </div>
        </div>
        <div
          class="am-steps-main"
        >
          <div
            class="am-steps-title"
          >
            Step 4
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`renders ./components/steps/demo/horizontal.md correctly 1`] = `
<div
  class="am-wingblank am-wingblank-lg stepsExample"
>
  <div
    class="sub-title"
  >
    Horizontal small size
  </div>
  <div
    class="am-whitespace am-whitespace-md"
  />
  <div
    class="am-steps am-steps-small am-steps-horizontal am-steps-label-vertical am-steps-hidden"
  >
    <div
      class="am-steps-item am-steps-status-finish am-steps-custom"
      style="margin-right:-1px;"
    >
      <div
        class="am-steps-tail"
        style="padding-right:1px;"
      >
        <i />
      </div>
      <div
        class="am-steps-step"
      >
        <div
          class="am-steps-head"
        >
          <div
            class="am-steps-head-inner"
          >
            <span
              class="am-steps-icon"
            >
              <svg
                class="am-icon am-icon-check-circle-o am-icon-md"
              >
                <use
                  xlink:href="#check-circle-o"
                />
              </svg>
            </span>
          </div>
        </div>
        <div
          class="am-steps-main"
        >
          <div
            class="am-steps-title"
          >
            Finished
          </div>
          <div
            class="am-steps-description"
          >
            This is description
          </div>
        </div>
      </div>
    </div>
    <div
      class="am-steps-item am-steps-status-process"
      style="margin-right:-1px;"
    >
      <div
        class="am-steps-tail"
        style="padding-right:1px;"
      >
        <i />
      </div>
      <div
        class="am-steps-step"
      >
        <div
          class="am-steps-head"
        >
          <div
            class="am-steps-head-inner"
          >
            <span
              class="am-steps-icon"
            >
              2
            </span>
          </div>
        </div>
        <div
          class="am-steps-main"
        >
          <div
            class="am-steps-title"
          >
            In Progress
          </div>
          <div
            class="am-steps-description"
          >
            This is description
          </div>
        </div>
      </div>
    </div>
    <div
      class="am-steps-item am-steps-status-wait am-steps-custom ellipsis-item"
    >
      <div
        class="am-steps-tail"
        style="padding-right:0;"
      >
        <i />
      </div>
      <div
        class="am-steps-step"
      >
        <div
          class="am-steps-head"
        >
          <div
            class="am-steps-head-inner"
          >
            <span
              class="am-steps-icon"
            >
              <svg
                class="am-icon am-icon-ellipsis am-icon-md"
              >
                <use
                  xlink:href="#ellipsis"
                />
              </svg>
            </span>
          </div>
        </div>
        <div
          class="am-steps-main"
        >
          <div
            class="am-steps-title"
          >
            Waiting
          </div>
          <div
            class="am-steps-description"
          >
            This is description
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="sub-title"
  >
    Horizontal default size
  </div>
  <div
    class="am-whitespace am-whitespace-md"
  />
  <div
    class="am-steps am-steps-horizontal am-steps-label-vertical am-steps-hidden"
  >
    <div
      class="am-steps-item am-steps-status-finish am-steps-custom"
      style="margin-right:-1px;"
    >
      <div
        class="am-steps-tail"
        style="padding-right:1px;"
      >
        <i />
      </div>
      <div
        class="am-steps-step"
      >
        <div
          class="am-steps-head"
        >
          <div
            class="am-steps-head-inner"
          >
            <span
              class="am-steps-icon"
            >
              <svg
                class="am-icon am-icon-check-circle-o am-icon-md"
              >
                <use
                  xlink:href="#check-circle-o"
                />
              </svg>
            </span>
          </div>
        </div>
        <div
          class="am-steps-main"
        >
          <div
            class="am-steps-title"
          >
            Finished
          </div>
          <div
            class="am-steps-description"
          >
            This is description
          </div>
        </div>
      </div>
    </div>
    <div
      class="am-steps-item am-steps-status-process"
      style="margin-right:-1px;"
    >
      <div
        class="am-steps-tail"
        style="padding-right:1px;"
      >
        <i />
      </div>
      <div
        class="am-steps-step"
      >
        <div
          class="am-steps-head"
        >
          <div
            class="am-steps-head-inner"
          >
            <span
              class="am-steps-icon"
            >
              2
            </span>
          </div>
        </div>
        <div
          class="am-steps-main"
        >
          <div
            class="am-steps-title"
          >
            In Progress
          </div>
          <div
            class="am-steps-description"
          >
            This is description
          </div>
        </div>
      </div>
    </div>
    <div
      class="am-steps-item am-steps-status-wait am-steps-custom ellipsis-item"
    >
      <div
        class="am-steps-tail"
        style="padding-right:0;"
      >
        <i />
      </div>
      <div
        class="am-steps-step"
      >
        <div
          class="am-steps-head"
        >
          <div
            class="am-steps-head-inner"
          >
            <span
              class="am-steps-icon"
            >
              <svg
                class="am-icon am-icon-ellipsis am-icon-md"
              >
                <use
                  xlink:href="#ellipsis"
                />
              </svg>
            </span>
          </div>
        </div>
        <div
          class="am-steps-main"
        >
          <div
            class="am-steps-title"
          >
            Waiting
          </div>
          <div
            class="am-steps-description"
          >
            This is description
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="sub-title"
  >
    Horizontal customized icon
  </div>
  <div
    class="am-whitespace am-whitespace-md"
  />
  <div
    class="am-steps am-steps-horizontal am-steps-label-vertical am-steps-hidden"
  >
    <div
      class="am-steps-item am-steps-status-process am-steps-custom error-tail"
      style="margin-right:-1px;"
    >
      <div
        class="am-steps-tail"
        style="padding-right:1px;"
      >
        <i />
      </div>
      <div
        class="am-steps-step"
      >
        <div
          class="am-steps-head"
        >
          <div
            class="am-steps-head-inner"
          >
            <span
              class="am-steps-icon"
            >
              <svg
                class="am-icon am-icon-pay-circle am-icon-md"
              >
                <use
                  xlink:href="#pay-circle"
                />
              </svg>
            </span>
          </div>
        </div>
        <div
          class="am-steps-main"
        >
          <div
            class="am-steps-title"
          >
            Step 1
          </div>
        </div>
      </div>
    </div>
    <div
      class="am-steps-item am-steps-status-error am-steps-custom"
      style="margin-right:-1px;"
    >
      <div
        class="am-steps-tail"
        style="padding-right:1px;"
      >
        <i />
      </div>
      <div
        class="am-steps-step"
      >
        <div
          class="am-steps-head"
        >
          <div
            class="am-steps-head-inner"
          >
            <span
              class="am-steps-icon"
            >
              <svg
                class="am-icon am-icon-pay-circle am-icon-md"
              >
                <use
                  xlink:href="#pay-circle"
                />
              </svg>
            </span>
          </div>
        </div>
        <div
          class="am-steps-main"
        >
          <div
            class="am-steps-title"
          >
            Step 2
          </div>
        </div>
      </div>
    </div>
    <div
      class="am-steps-item am-steps-status-wait am-steps-custom"
    >
      <div
        class="am-steps-tail"
        style="padding-right:0;"
      >
        <i />
      </div>
      <div
        class="am-steps-step"
      >
        <div
          class="am-steps-head"
        >
          <div
            class="am-steps-head-inner"
          >
            <span
              class="am-steps-icon"
            >
              <svg
                class="am-icon am-icon-pay-circle am-icon-md"
              >
                <use
                  xlink:href="#pay-circle"
                />
              </svg>
            </span>
          </div>
        </div>
        <div
          class="am-steps-main"
        >
          <div
            class="am-steps-title"
          >
            Step 3
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="sub-title"
  >
    Horizontal timeline mode
  </div>
  <div
    class="am-whitespace am-whitespace-md"
  />
  <div
    class="am-steps am-steps-small am-steps-horizontal am-steps-label-vertical am-steps-hidden timeline"
  >
    <div
      class="am-steps-item am-steps-status-wait am-steps-custom"
      style="margin-right:-1px;"
    >
      <div
        class="am-steps-tail"
        style="padding-right:1px;"
      >
        <i />
      </div>
      <div
        class="am-steps-step"
      >
        <div
          class="am-steps-head"
        >
          <div
            class="am-steps-head-inner"
          >
            <span
              class="am-steps-icon"
            >
              <span
                class="fake-icon"
              />
            </span>
          </div>
        </div>
        <div
          class="am-steps-main"
        >
          <div
            class="am-steps-title"
          >
            Now
          </div>
          <div
            class="am-steps-description"
          >
            Buy
          </div>
        </div>
      </div>
    </div>
    <div
      class="am-steps-item am-steps-status-wait am-steps-custom"
      style="margin-right:-1px;"
    >
      <div
        class="am-steps-tail"
        style="padding-right:1px;"
      >
        <i />
      </div>
      <div
        class="am-steps-step"
      >
        <div
          class="am-steps-head"
        >
          <div
            class="am-steps-head-inner"
          >
            <span
              class="am-steps-icon"
            >
              <span
                class="fake-icon"
              />
            </span>
          </div>
        </div>
        <div
          class="am-steps-main"
        >
          <div
            class="am-steps-title"
          >
            Nov. 3rd
          </div>
          <div
            class="am-steps-description"
          >
            Buy success
          </div>
        </div>
      </div>
    </div>
    <div
      class="am-steps-item am-steps-status-wait am-steps-custom"
    >
      <div
        class="am-steps-tail"
        style="padding-right:0;"
      >
        <i />
      </div>
      <div
        class="am-steps-step"
      >
        <div
          class="am-steps-head"
        >
          <div
            class="am-steps-head-inner"
          >
            <span
              class="am-steps-icon"
            >
              <span
                class="fake-icon"
              />
            </span>
          </div>
        </div>
        <div
          class="am-steps-main"
        >
          <div
            class="am-steps-title"
          >
            Nov. 4th
          </div>
          <div
            class="am-steps-description"
          >
            Profit arrival
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
